gcloud/datastore/test_dataset.py

```diff
@@ -157,6 +157,11 @@ def allocate_ids(self, dataset_id, key_pbs):
         num_pbs = len(key_pbs)
         return [_KeyProto(i) for i in range(num_pbs)]
 
+    def delete_entities(self, dataset_id, key_pbs):
```

Comment:
I was just trying to bail water out of the boat with these tests.

This class gets moved in #483